incubator/airflow/README.md

```diff
@@ -0,0 +1,127 @@
+
+Both can be placed either at the root of a domain or at a sub path, for example:
+
+```
+http://mycompany.com/airflow/
+http://mycompany.com/airflow/flower
+```
+
+NOTE: Mounting the Airflow UI under a subpath requires an airflow version > 1.9.x. For the moment
+(March 2018) this is **not** available on official package, you will have to use an image where
+airflow has been updated to its current HEAD. You can use the following one:
+`stibbons31/docker-airflow-dev:2.0dev`
+
+Please also note than Airflow UI and Flower do not behave the same:
+
+- Airflow Web UI behave transparently, to configure it one just need to specify the
+  `ingress.web.path` value.
+- Flower cannot handle this scheme directly and requires to use an URL rewrite mechanism in front
+  of it. In short, it is able to generate the right URLs in the returned HTML file but cannot
+  respond to these URL. It is commonly found in software that wasn't intended to work under
+  something else than a root URL or localhost port. To use it, see the `value.yaml` in detail on how
+  to configure your ingress controller to rewrite the URL (or "strip" the prefix path).
+
+### Airflow configuration
+
+`airflow.cfg` configuration can be changed by defining environment variables in the following form:
+`AIRFLOW__<section>__<key>`.
+
+See the
+[Airflow documentation for more information](http://airflow.readthedocs.io/en/latest/configuration.html?highlight=__CORE__#setting-configuration-options)
+
+This helm chart allows you to add these additional settings with the value key `airflow.config`.
+
+### Worker Statefulset
+
+Celery workers uses StatefulSet instead of deployment.
+It is used to freeze their DNS using a Kubernetes Headless Service, and allow the webserver to
+requests the logs from each workers individually.
+This requires to expose a port (8793) and ensure the pod DNS is accessible to the web server pod,
+which is why StatefulSet is for.
+
+## DAGs Deployment
+
+### Mount a Persistent Volume
+
+You can store your DAG files on an external volume, and mount this volume into the relevant Pods
+(scheduler, web, worker). In this scenario, your CI/CD pipeline should update the DAG files in the 
+PV.
+Since all Pods should have the same collection of DAG files, it is recommended to create just one PV
+that is shared. This ensures that the Pods are always in sync about the DagBag. 
+
+To share a PV with multiple Pods, the PV needs to have accessMode 'ReadOnlyMany' or 'ReadWriteMany'.
+If you are on AWS, you can use [Elastic File System (EFS)](https://aws.amazon.com/efs/). 
+If you are on Azure, you can use [Azure File Storage (AFS)](https://docs.microsoft.com/en-us/azure/aks/azure-files-dynamic-pv).
+
+
+### Embedded DAGs
+
+If you want more control on the way you deploy your DAGs, you can use embedded DAGs, where DAGs
+are burned inside the Docker container deployed as Scheduler and Workers.
+
+Be aware this requirement more heavy tooling than using git-sync, especially if you use CI/CD:
```

Comment:
This is the only mention fo git-sync in the readme, and it doesn't state how to use it. I see it in the values.yaml file, but nowhere else. Is it supported or not? If so, can instructions be added for how to use it? 

I assume you would set `git_sync_enabled` to `true`, but then where do I put the dags in the repository? If its a private repository, how would I set up access?

Reply:
hi. I did not maintained this part of the chart, so git-sync is no more used. You can try to port integrate it properly in kubernetes (see https://github.com/kubernetes/git-sync/blob/master/demo/config/pod.yaml#L9)